### PR TITLE
legacy content view test support added

### PR DESCRIPTION
**Pull request by omkarkhatavkar**
As ContentView has been changed to new UI, we are still running tests to legacy ui. To support testing legacy UI, some refactor in code was required. This PR helps to fix that.

### CODE REVIEW

**Review by latran — APPROVED** [review]
Very nice! This fix was easier than I thought.  I also ran a few tests on my local and it navigates correctly.  Ack!

**Review by jyejare — COMMENTED** [review]
ACK pending curious question !

File: airgun/entities/contentview.py [hunk 1]
```diff
@@ -232,7 +234,10 @@ class ShowAllContentViews(NavigateStep):
 
     @retry_navigation
     def step(self, *args, **kwargs):
```
[the remark this answers is not in the document]
Comment (omkarkhatavkar) [reply airgun/entities/contentview.py:236]: @latran is working on a new view that will be configurable from tests which going to select a new content view. Basically, this fix will help to navigate to old content view.

[the remark this answers is not in the document]
Comment (latran) [reply airgun/entities/contentview.py:236]: @jyejare The new content view is it's own thing.  I'm currently working on creating a new entities and view files for the new content view.

[the remark this answers is not in the document]
Comment (omkarkhatavkar) [reply airgun/entities/contentview.py:236]: @jyejare here is the PR for that https://github.com/SatelliteQE/airgun/pull/656

[the remark this answers is not in the document]
Comment (jyejare) [reply airgun/entities/contentview.py:236]: Ok Cool!

**Review by jyejare — APPROVED** [review]

**Pull request merged**